src/styles/sidebar.styl

```diff
@@ -1,19 +1,19 @@
-.con-wrapper
+.con-wrapper // @stylint ignore
```

Comment:
I have another `.con-wrapper` selector here https://github.com/cozy/cozy-data-connect/blob/master/app/styles/app.styl#L1. I prefered to keep these codes split for now, so I put the magical comment. But all that should with css-modules usage :)